src/triggers/Bank_Account_Trigger.trigger

```diff
@@ -0,0 +1,30 @@
+/**
+    Description :   Bank_Account_Trigger for trigger an sObject
+                    Inserting Account Number
+                    Sending Mail after creating Account
+                    Sending Mail after Deactivating Account
+    Created BY :    Aryan Sharma
+    Created Date :  20-07-2022
+    Revision Logs : V_1.0 - Created 								  
+**/
+trigger Bank_Account_Trigger on Bank_Account__c (before insert, after insert,after delete) {
+    if(Trigger.isBefore) {
+        if(Trigger.isInsert) {
+            for(Bank_Account__c accNewList : Trigger.New) { 
```

Comment:
why are we calling method in loop ?

Reply:
Right method will not called in a loop.
I left loop here because in beginning I used to go through only Trigger 
then after I go through the helper Class.